datadog/api/metrics.py

```diff
@@ -54,6 +54,12 @@ def send(cls, metrics=None, attach_host_name=True, **single_metric):
         :param metric: the name of the time series
         :type metric: string
 
+        :param single_metric: a list of metric names
+        :type metric: list
```

Comment:
That's not it. `single_metric` is just name of the kwargs argument, but it should just consist of 5 arguments (some of which are optional): `metric`, `points`, `host`, `tags` and `type`, which are documented just below.
`metrics` should be a list of dictionaries that follow the `single_metric` format, i.e. dict containing the same 5 keys than `single_metric`.

In any case, `single_metric` should not be documented as is, we already document each keyword argument that it should contain